fmlloader/src/main/java/net/minecraftforge/fml/loading/targets/CommonDevLaunchHandler.java

```diff
@@ -77,11 +77,26 @@ protected String[] preLaunch(String[] arguments, ModuleLayer layer) {
     protected List<Path> getFmlStuff(String[] classpath) {
         // We also want the FML things, fmlcore, javafmllanguage, mclanguage, I don't like hard coding these, but hey whatever works for now.
         return Arrays.stream(classpath)
-            .filter(e -> e.contains("fmlcore") || e.contains("javafmllanguage") || e.contains("lowcodelanguage") || e.contains("mclanguage"))
+            .filter(e -> {
+                // Extract file name from path
+                String name = e.substring(e.lastIndexOf(File.separatorChar) + 1);
```

Comment:
This file name extraction code could be moved in an utility method in `FileUtils`

Reply:
done